Swiftfin tvOS/Views/VideoPlayer/NativeVideoPlayer.swift

```diff
@@ -37,7 +37,7 @@ struct NativeVideoPlayer: View {
             if let _ = videoPlayerManager.currentViewModel {
                 playerView
             } else {
-//                VideoPlayer.LoadingView()
+                // TODO: Implement more polished loading screen.
```

Comment:
To me, the commented out view is better because that's the actual view that needs to be implemented. (see the file/view)